pkgs/os-specific/linux/nvidia-x11/default.nix

```diff
@@ -19,8 +19,10 @@ rec {
   # Policy: use the highest stable version as the default (on our master).
   stable = if stdenv.hostPlatform.system == "x86_64-linux"
     then generic {
-      version = "460.73.01";
-      sha256_64bit = "120ymf59l6nipczszf82lrm2p4ihhqyv2pfwwfg9wy96vqcckc8i";
+      version = "460.80";
+      persistencedVersion = "460.73.01";
```

Comment:
There were problems once building a different version of  passtrhough nvidia-settings https://github.com/NixOS/nixpkgs/pull/108795. We should test if this works.

Reply:
This should have a synchronized version pushed to the GitHub repository within 24 hours usually.

Reply:
Yeah, maybe we should wait a bit...